webpay/src/Utils/TransbankSdkOneclick.php

```diff
@@ -155,20 +145,15 @@ public function authorizeTransaction($username, $tbkUser, $parentBuyOrder, $chil
                     'installments_number' => 1,
                 ],
             ];
-
             $resp = $this->transaction->authorize($username, $tbkUser, $parentBuyOrder, $details);
-            $this->log->logInfo('authorizeTransaction - resp: ' . json_encode($resp));
+            $this->log->logInfo('authorize - resp: ' . json_encode($resp));
             return $resp;
         } catch (Exception $e) {
-            $result = [
-                'error'  => 'Error al autorizar el pago',
-                'detail' => $e->getMessage(),
-            ];
-            $this->log->logError(json_encode($result));
+            $errorMessage = "Error al autorizar el pago para => userName: {$username}, buyOrder: {$parentBuyOrder}, error: {$e->getMessage()}";
+            $this->log->logError($errorMessage);
+            throw new EcommerceException($errorMessage, 0, $e);
```

Comment:
replace this magic number

Reply:
added constructor with default code exception